cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java

```diff
@@ -323,22 +323,22 @@ public JmsProvider getJmsProvider() {
    */
   public static class JmsPluginConfig extends PluginConfig {
     @Name(JMS_DESTINATION_NAME)
-    @Description("Name of the destination to get messages")
+    @Description("Name of the destination from which to get messages.")
     private String destinationName;
 
     @Name(JMS_MESSAGES_TO_RECEIVE)
-    @Description("Max number messages should be retrieved per poll. The default value is 50.")
+    @Description("Max number of messages that should be retrieved per poll. The default value is 50.")
     @Nullable
     private Integer messagesToReceive;
 
     @Name(JMS_NAMING_FACTORY_INITIAL)
     @Description("The fully qualified class name of the factory class that will create an initial context. " +
-      "This will be passed to JNDI initial context as " + Context.INITIAL_CONTEXT_FACTORY)
+      "This will be passed to the JNDI initial context as " + Context.INITIAL_CONTEXT_FACTORY + ".")
```

Comment:
Should these statics be in enclosed in single quotes, as below?
What I mean is:
```
"This will be passed to the JNDI initial context as '" + Context.INITIAL_CONTEXT_FACTORY + "'.")
````
like below:
```
"for the JMS plugin. Defaults to 'JMSProvider'.")
```